hoge0516mvc/wwwroot/Content/Site.css

```
@@ -0,0 +1,20 @@
+﻿/* Set padding to keep content from hitting the edges */
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)